src/Core/RevEng.Core.60/Routines/Functions/SqlServerFunctionModelFactory.cs

```diff
@@ -21,19 +21,10 @@ public SqlServerFunctionModelFactory()
     ROUTINE_NAME,
     CAST(CASE WHEN (DATA_TYPE != 'TABLE') THEN 1 ELSE 0 END AS bit) AS IS_SCALAR
 FROM INFORMATION_SCHEMA.ROUTINES
+LEFT JOIN sys.extended_properties AS [ep] on [ep].major_id = object_id(QUOTENAME(ROUTINE_SCHEMA) + '.' + QUOTENAME(ROUTINE_NAME)) AND [ep].minor_id = 0 AND [ep].class = 1
```

Comment:
Where is the name criteria??

Reply:
what do you mean ?

Reply:
wrap it with [] ?

Reply:
wanted to keep in same style like the other table

Reply:
I mean the name = 'microsoft_toolsxxx' criteria

Reply:
fixed